ast/node.go

```diff
@@ -76,6 +83,8 @@ const (
 	NodeUnaryExpr
 	NodeMemberExpr
 	NodeCallExpr
+	NodeVarDecl
+	NodeVarDecls
```

Comment:
```
if(var a = 1, b = 2, c =3) console.log("ok")
VM85:1 Uncaught SyntaxError: Unexpected token var
```

Reply:
I don't tried to use var statements as expressions, they are being accepted right now as a bug ? I just added the var parser on the node parser, I did not realized that the parseNode function is the expression parser  (it is the main entrypoint no ?)

Reply:
see my other comment. Sorry for not being clear.